processor/transform/handle-missing-components.ts

```diff
@@ -0,0 +1,31 @@
+import type { Parents } from 'mdast';
+import type { Transform } from 'mdast-util-from-markdown';
+import type { MdxJsxFlowElement, MdxJsxTextElement } from 'mdast-util-mdx';
+
+import { visit } from 'unist-util-visit';
+
+import * as Components from '../../components';
+import { isMDXElement } from '../utils';
+
+const handleMissingComponents =
```

Comment:
(Will this also "fix" RM-12236 right? Not in the long term, obviously, but at least in view mode for now I'd imagine?)

Reply:
I suppose, but probably a better short term fix would be to bump the page size?

Reply:
lol yes that's definitely better!